docs/content/documentation/content/shortcodes.md

```diff
@@ -169,3 +169,30 @@ Usage example:
 Result example:
 
 {{ gist(url="https://gist.github.com/Keats/e5fb6aad409f28721c0ba14161644c57") }}
+
+### Twitch
+Embed a Twitch stream with or without chat.
+
+The arguments are:
+
+- `channel`: the channel name (mandatory)
+- `chat`: shows the stream chat next to the video embed 
+- `class`: a class to add the `div` surrounding the iframe
+- `videoWidth`: the px **width** the video embed should have 
+- `videoHeight`: the px **height** the video embed should have
+- `chatWidth`: the px **width** the chat embed should have 
+- `chatHeight`: the px **height** the chat embed should have
```

Comment:
when we only get one iframe, only one set of (height,width) will be required

Reply:
Sure, I will change it